api/HTMLElement.json

```diff
@@ -416,6 +463,53 @@
           }
         }
       },
+      "autofocus": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "79"
```

Comment:
Autofocus behavior is very old. Can you check if we have other "autofocus" entries anywhere in BCD and what their data is?

Reply:
Yeah, I can dig into this more.  My best guess is that it may be due to the `autofocus` attribute being defined on the individual elements (or on the `Element` API), and then moved to `HTMLElement`?  The collector tests `HTMLElement` specifically.

Reply:
Yeah, it might have been on HTMLInputElement originally, that's worth checking.

Reply:
Okay, so it looks like they were initially defined on individual HTML elements (`button`, `input`, `select`, and `textarea`), and then in [Chrome 79](https://storage.googleapis.com/chromium-find-releases-static/0fb.html#0fbf9bdd80bfef802dcc57ab0c295a95a3fa39b3), they were moved to the `HTMLOrForeignElement` mixin.  Should we add a note as such?

Reply:
Sounds like this is similar to https://github.com/mdn/browser-compat-data/issues/7519 and `partial_implementation` is appropriate. Problem is figuring out what the note should say, we need to figure out all the places that autofocus can work, and if any of those weren't supported from the beginning.

I half expect that the actual support hasn't changes and that it's only the IDL that's changed, so that there won't be a note that makes sense to web developers. But testing first makes sense.

Reply:
In order to unblock the other half of this PR, I've filed https://github.com/mdn/browser-compat-data/issues/8745 and will revert this bit and land this PR.